files/zh-cn/learn/css/css_layout/normal_flow/index.md

```diff
@@ -57,7 +57,7 @@ slug: Learn/CSS/CSS_layout/Normal_Flow
 
 <p>We are separated by our margins. Because of margin collapsing, we are separated by the width of one of our margins, not both.</p>
 
-<p>inline elements <span>like this one</span> and <span>this one</span> sit on the same line as one another, and adjacent text nodes, if there is space on the same line. Overflowing inline elements will <span>wrap onto a new line if possible (like this one containing text)</span>, or just go on to a new line if not, much like this image will do: <img src="/en-US/docs/Learn/CSS/CSS_layout/Normal_Flow/long.jpg" alt="snippet of cloth" /></p>
+<p>inline elements <span>like this one</span> and <span>this one</span> sit on the same line as one another, and adjacent text nodes, if there is space on the same line. Overflowing inline elements will <span>wrap onto a new line if possible (like this one containing text)</span>, or just go on to a new line if not, much like this image will do: <img src="/zh-CN/docs/Learn/CSS/CSS_layout/Normal_Flow/long.jpg" alt="snippet of cloth" /></p>
```

Comment:
This image is only available in en-US, as mdn/yari#5652 is not fixed. Pointing to `en-US` as a workaround here.

Reply:
Actually, in another locale, I was able to specify `long.jpg` with no issues, so I think that issue may actually be resolved!

<img width="526" alt="image" src="https://github.com/mdn/translated-content/assets/5179191/3e4b438b-5e62-4eb0-b960-086e5b30f53b">